Library/Formula/postgresql.rb

```diff
@@ -72,13 +74,18 @@ def install
       ENV.append %w{CFLAGS LDFLAGS}, "-arch #{Hardware::CPU.arch_32_bit}"
     end
 
+    system "pkill postgres || true"
     system "./configure", *args
     system "make", "install-world"
   end
 
   def post_install
-    unless File.exist? "#{var}/postgres"
-      system "#{bin}/initdb", "#{var}/postgres"
+    if (var/"postgres").exist?
+      mv var/"postgres", var/"postgres-#{version}"
```

Comment:
That's pretty niche, surely?

If it's really that likely to happen, I guess we could ` exit 1 ` here and print an error message "Remove x directory/Move x directory out of the way, blah" etc?